### PR TITLE
createController

### PR DESCRIPTION
**Pull request by elenmohsen**
controller

### CODE REVIEW

File: app/Http/Controllers/ExampleController.php [hunk 1]
```diff
@@ -6,5 +6,26 @@
...
+    function cv(){
+        return view('cv');
+    }
+
+    function contactus(){
+        return view('contactus');
+    }
+
+    function personaldata(Request $request){
+        $name= $request->name;
```
Comment (Nora-Lootfy) [inline app/Http/Controllers/ExampleController.php]: nice work ya Ellen,,

your code is very neat

**Review by Nora-Lootfy — APPROVED** [review]
I have no comments,, you nailed it

File: app/Http/Controllers/ExampleController.php (hunk 1 above)
Comment (elenmohsen) [reply app/Http/Controllers/ExampleController.php]: ok thanks

**Review by Nora-Lootfy — APPROVED** [review]
nice work ya Elen, no comments😊

**Pull request merged**